src/esn.calendar.libs/app/services/shells/calendar-right-shell.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-/* global chai, __FIXTURES__, _: false */
+/* global chai, __FIXTURES__, _: true */
```

Comment:
Why?

Reply:
In a desperate moment :trollface: Have reverted it. Thanks!